docs/tutorial/devtools-extension.md

```diff
@@ -8,6 +8,8 @@ the `BrowserWindow.addDevToolsExtension` API to load them. The loaded extensions
 will be remembered so you don't need to call the API every time when creating
 a window.
 
+## NOTE: React DevTools does not work, follow the issue here https://github.com/atom/electron/issues/915
```

Comment:
I'm 👍 on this, but showing it as a title is probably too strong, maybe we can just show it in bold:

```markdown
** Note: ... **
```